ministryofjustice/correspondence_tool_public
# CT-3129 Add cronjob for redundant image deletion

## Description


## Self-review checklist


### Screenshots
Making sure no non-master images newer than 14 days deleted:
![image](https://user-images.githubusercontent.com/22935203/97303097-da5ede00-1851-11eb-80e7-19be5056e576.png)

Making sure no more than 10 master images are deleted, well there's only one:
![image](https://user-images.githubusercontent.com/22935203/97303702-a59f5680-1852-11eb-8ced-5961146d92a7.png)

Make sure that correct images were deleted:
![image](https://user-images.githubusercontent.com/22935203/97303773-bfd93480-1852-11eb-9389-b007cee45e1f.png)

So since there's only a few images, I tried to call all images on the server, to see if any should have been picked up and deleted. 

According to this there are only 26 images - so there's not 100s left so that's good: 
![image](https://user-images.githubusercontent.com/22935203/97305416-f617b380-1854-11eb-8b1f-3696ba284c8a.png)

Out of non-master images left, there should be nothing older than 14 days, which there are not (nulls):
![image](https://user-images.githubusercontent.com/22935203/97305729-6cb4b100-1855-11eb-9cc6-01232199fb3b.png)


Out of master images left there should be no more than 10, and it is exactly 10
![image](https://user-images.githubusercontent.com/22935203/97306340-43e0eb80-1856-11eb-994d-45e7a2a74887.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3129

### Deployment
deployment file changed, so deployment should be tested, and if production, Cronjob should be applied

### Manual testing instructions
Check screenshots for script. Also once applied to prod, check if cronjob is there by doing:
kubectl get cronjobs -n contact-moj-production
